src/core/base.jl

```diff
@@ -176,6 +177,7 @@ function _ref_add_core!(nw_refs::Dict{Int,<:Any}; base_length=5000.0, base_press
         for (idx, compressor) in ref[:compressor]
             i = compressor["fr_junction"]
             j = compressor["to_junction"]
+            compressor["area"] = pi * compressor["diameter"]  * compressor["diameter"] / 4.0
```

Comment:
I like adding area calculations.  We should now use this directly in other parts of the code (i.e. the resistance calculations) rather than recompute

Reply:
@rb004f converted this.